test-data/unit/check-classes.test

```diff
@@ -4131,6 +4131,18 @@ class C(str, str):
 [out]
 main:1: error: Duplicate base class "str"
 
+[case testDupBaseClassesIsNotBlocking]
+class A:
+    def method(self) -> str: ...
+
+class B(A, A):  # E: Duplicate base class "A"
+    ...
+
+b: B
+# We use dummy type info, so no real attrs will be there:
```

Comment:
That's pretty unfortunate, can we make it inherit from Any or just skip the duplicate base instead?

Reply:
Yeap, got it. Fixed!